dataland-frontend/src/components/pages/AdminAllRequestsOverview.vue

```diff
@@ -322,19 +334,22 @@ export default defineComponent({
      * Resets selected frameworks and searchBarInput
      */
     resetFilterAndSearchBar() {
+      this.currentChunkIndex = 0;
       this.selectedFrameworks = [];
       this.selectedRequestStatus = [];
       this.searchBarInput = '';
     },
 
     /**
-     * Updates the current Page in the parent component
+     * Updates the current Page
      * @param event DataTablePageEvent
      */
     onPage(event: DataTablePageEvent) {
       window.scrollTo(0, 0);
-      if (event.page != this.currentPage) {
-        this.currentPage = event.page;
+      if (event.page != this.currentChunkIndex) {
+        this.currentChunkIndex = event.page;
+        this.firstRowIndex = this.currentChunkIndex * this.rowsPerPage;
```

Comment:
Does this mean if i go to the second chunk (chunkIndex = 2) that the firstRowIndex is = 200?